AskMeAnything/handler.ts

```
@@ -0,0 +1,39 @@
+  if (request.method === 'GET') {
+    context.log(`Http request on GET for ${request.url}`)
+    return {
+      body: JSON.stringify({ data: `Hello World` }),
+      status: 200,
+    }
+  } else {
+    const body = await request.text()
+    const contentType = readHeader(request, 'content-type')
+    const payload = parseBody(body, contentType)
+
+    if (payload && payload.challenge) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)